tests/php/Shortcodes/FileLinkTrackingTest.php

```diff
@@ -33,8 +33,8 @@ protected function setUp(): void
         $files = File::get()->exclude('ClassName', Folder::class);
         foreach ($files as $file) {
             // Mock content for files
-            $content = $file->Filename . ' ' . str_repeat('x', 1000000);
-            $file->setFromString($content, $file->Filename);
+            $sourcePath = __DIR__ . '/FileLinkTrackingTest/' . $file->Name;
+            $file->setFromLocalFile($sourcePath, $file->Filename);
```

Comment:
Had to have a real image to avoid the following error:
> ImagickException: negative or zero image size `/var/www/html/silverstripe-cache/user/interventionimage_rcPBnB.jpg' @ error/image.c/CloneImage/794